lib/nfsbase.pm

```diff
@@ -0,0 +1,146 @@
+package nfsbase;
```

Comment:
i would suggest to rename it to nfs_common or something as it is not really used as a parent class anywhere

Reply:
I'd vote for converting this to a base class, wether done now or later it's a different thing :). @pdostal What's your take? Wanna try going OOPerl? :), or would you prefer to rename to nfs_common (file and the package) for the time being?

Reply:
I renamed it to `nfs_common` for now, I'm still not that familiar with Perl and I use the `y2_module_consoletest` base class.